dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPI.java

```diff
@@ -187,6 +188,9 @@ Optional<Rule> getRule(final Experiment experiment)
     ExperimentResults getResults(final Experiment experiment, User user)
             throws DotDataException, DotSecurityException;
 
+    @CloseDBIfOpened
```

Comment:
not needed on the interface

Reply:
doc?

Reply:
Not mine but removed it anyways @jdotcms